<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/resources/resources/manager.png</normaloff>:/new/resources/resources/manager.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutNotes">
        <item>
         <widget class="QLabel" name="labelNotes">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <column>
           <property name="text">
            <string>Folder1</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Folder1</string>
           </property>
           <item>
            <property name="text">
             <string>file1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>audio1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>video1</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Folder2</string>
           </property>
           <item>
            <property name="text">
             <string>audio2</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>file3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>audio3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>video3</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonSave">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/new/resources/resources/save.png</normaloff>:/new/resources/resources/save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/new/resources/resources/delete.png</normaloff>:/new/resources/resources/delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRestore">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Restore</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/new/resources/resources/restore.png</normaloff>:/new/resources/resources/restore.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutTags">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>100</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTags">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetTags">
          <property name="autoScroll">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>toto</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tata</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>titi</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTag">
          <property name="statusTip">
           <string>add tag </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonAddTag">
            <property name="text">
             <string>add tag</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/new/resources/resources/addtag.png</normaloff>:/new/resources/resources/addtag.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRemoveTag">
            <property name="text">
             <string>remove tag</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/new/resources/resources/removetag.png</normaloff>:/new/resources/resources/removetag.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidgetOnglets">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Editor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Editor</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Text">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Text</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Html">
       <attribute name="title">
        <string>Html</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Latex">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Latex</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewArticle"/>
     <addaction name="actionNewDocument"/>
     <addaction name="actionNewPicture"/>
     <addaction name="actionNewAudio"/>
     <addaction name="actionNewVideo"/>
    </widget>
    <widget class="QMenu" name="menuTaper_ici">
     <property name="title">
      <string>Taper ici</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpenArticle"/>
     <addaction name="actionOpenDocument"/>
     <addaction name="actionPicture"/>
     <addaction name="actionOpenAudio"/>
     <addaction name="actionOpenVideo"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuOpen"/>
    <addaction name="actionExit_2"/>
    <addaction name="menuTaper_ici"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWorkspace"/>
   <addaction name="actionTrash"/>
   <addaction name="separator"/>
   <addaction name="actionNewArticle"/>
   <addaction name="actionNewDocument"/>
   <addaction name="actionNewPicture"/>
   <addaction name="actionNewAudio"/>
   <addaction name="actionNewVideo"/>
   <addaction name="separator"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionExit_2"/>
  </widget>
  <action name="actionNewArticle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/bluefile.png</normaloff>:/new/resources/resources/bluefile.png</iconset>
   </property>
   <property name="text">
    <string>Article</string>
   </property>
   <property name="toolTip">
    <string>new article</string>
   </property>
  </action>
  <action name="actionNewDocument">
   <property name="icon">
    <iconset>
     <disabledoff>:/new/resources/resources/folder.png</disabledoff>
    </iconset>
   </property>
   <property name="text">
    <string>Document</string>
   </property>
   <property name="toolTip">
    <string>new document</string>
   </property>
  </action>
  <action name="actionNewAudio">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/audio.png</normaloff>:/new/resources/resources/audio.png</iconset>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
   <property name="toolTip">
    <string>new audio</string>
   </property>
  </action>
  <action name="actionNewVideo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/video.png</normaloff>:/new/resources/resources/video.png</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
   <property name="toolTip">
    <string>new video</string>
   </property>
  </action>
  <action name="actionTrash">
   <property name="icon">
    <iconset>
     <normalon>:/new/resources/resources/trash.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>trash</string>
   </property>
   <property name="toolTip">
    <string>trash</string>
   </property>
  </action>
  <action name="actionWorkspace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/workspace.png</normaloff>:/new/resources/resources/workspace.png</iconset>
   </property>
   <property name="text">
    <string>workspace</string>
   </property>
   <property name="toolTip">
    <string>open workspace</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/export.png</normaloff>:/new/resources/resources/export.png</iconset>
   </property>
   <property name="text">
    <string>export</string>
   </property>
   <property name="toolTip">
    <string>export </string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/delete.png</normaloff>:/new/resources/resources/delete.png</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
   <property name="toolTip">
    <string>delete</string>
   </property>
  </action>
  <action name="actionOpenArticle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/bluefile.png</normaloff>:/new/resources/resources/bluefile.png</iconset>
   </property>
   <property name="text">
    <string>Article</string>
   </property>
   <property name="toolTip">
    <string>new article</string>
   </property>
  </action>
  <action name="actionOpenDocument">
   <property name="icon">
    <iconset>
     <disabledoff>:/new/resources/resources/folder.png</disabledoff>
    </iconset>
   </property>
   <property name="text">
    <string>Document</string>
   </property>
   <property name="toolTip">
    <string>new document</string>
   </property>
  </action>
  <action name="actionOpenAudio">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/audio.png</normaloff>:/new/resources/resources/audio.png</iconset>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
   <property name="toolTip">
    <string>new audio</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/video.png</normaloff>:/new/resources/resources/video.png</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
   <property name="toolTip">
    <string>new video</string>
   </property>
  </action>
  <action name="actionNewPicture">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/picture.png</normaloff>:/new/resources/resources/picture.png</iconset>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </action>
  <action name="actionPicture">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/picture.png</normaloff>:/new/resources/resources/picture.png</iconset>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resources/resources/quit.png</normaloff>:/new/resources/resources/quit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
